src/test/kotlin/step2/CalculatorTest.kt

```diff
@@ -0,0 +1,36 @@
+package step2
+
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Test
+
+class CalculatorTest {
+    @Test
+    fun `덧셈 테스트`() {
+        var calculator = Calculator("2 + 3")
```

Comment:
```suggestion
        val calculator = Calculator("2 + 3")
```

calculator 를 var 이 아니라, val 로 선언하신 이유가 있는지 궁금합니다.